packages/@react-stately/virtualizer/src/Virtualizer.ts

```diff
@@ -133,7 +133,7 @@ export class Virtualizer<T extends object, V, W> {
    */
   keyAtPoint(point: Point): Key | null {
     let rect = new Rect(point.x, point.y, 1, 1);
-    let layoutInfos = this.layout.getVisibleLayoutInfos(rect);
+    let layoutInfos = rect.height === 0 ? [] : this.layout.getVisibleLayoutInfos(rect);
```

Comment:
Same thing could probably happen if the width was zero? maybe check `rect.area === 0` instead?